symbolic-symcache/src/format.rs

```diff
@@ -177,34 +185,44 @@ pub struct FileRecord {
 pub struct FuncRecord {
     /// Low bits of the address.
     pub addr_low: u32,
+
     /// High bits of the address.
     pub addr_high: u16,
-    /// The length of the function. A value of 0xffff indicates that the size is unknown.
+
+    /// The length of the function.
+    ///
+    /// A value of `0xffff` indicates that the size is unknown.
+    /// We cannot cache any useful information for functions containing no instructions, so
+    /// the length is always positive.
     pub len: NonZeroU16,
+
     /// The line record of this function.  If it fully overlaps with an inline the record could be
     /// `~0`.
     pub line_records: Seg<LineRecord, u16>,
+
```

Comment:
It's going to be very lonely being the only one who's not in love with blank lines 😄